test/aria/utils/ArrayTestCase.js

```diff
@@ -237,6 +237,46 @@ Aria.classDefinition({
                 result += value;
             }, this);
             this.assertTrue(result === "123459");
+        },
+
+        testEvery : function () {
+            // shortcut
+            var every = aria.utils.Array.every;
+
+            // doing stuff on empty array should do nothing
+            var count = 0;
+            every([], function () {
+                count++;
+            });
+            this.assertTrue(count === 0);
+
+            // forEach simple test
```

Comment:
`forEach` should be `every`, oversight after copy-paste always happens ;)

Anyway I think it's better to leave it as `simple test`, I don't see the point of repeating the name of the tested function here, it doesn't bring more information (and again, it's annoying when copy-pasting). My comment applies only to your code, no need to change the rest of the module.